dotCMS/src/main/java/com/dotmarketing/business/CacheLocator.java

```diff
@@ -436,6 +443,7 @@ Cachable create() {
 	      	case MultiTreeCache : return new MultiTreeCache();
 	      	case ApiTokenCache : return new ApiTokenCache();
 	      	case PortletCache : return new PortletCache();
+	      	case ESQueryCache : return new com.dotcms.content.elasticsearch.ESQueryCache();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'com.dotcms.content.elasticsearch.ESQueryCache' due to existing import 'com.dotcms.content.elasticsearch.ESQueryCache'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5288908)